src/Collector/ValueObject/CollectedForm.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\Collector\ValueObject;
+
+/**
+ * @phpstan-import-type CollectedFormFieldArray from CollectedFormField
+ * @phpstan-type CollectedFormArray array{className: class-string, methodName: string, name: string, formFields: CollectedFormFieldArray[]}
+ */
+final class CollectedForm
```

Comment:
not all Components have fields, I think it is Form-specific

Reply:
Well it could be optional property in Component. I would prefer to treat it as extended component - it would simplify code because we would not have to duplicate very similar collection/storing/finding with froms and components and passing them around etc. it owuld be just special version of Component